BUILD.bazel

```diff
@@ -1513,20 +1529,20 @@ cc_test(
     ],
 )
 
-# cc_test(
-#     name = "gcs_actor_scheduler_mock_test",
-#     size = "small",
-#     srcs = [
-#         "src/ray/gcs/gcs_server/test/gcs_actor_scheduler_mock_test.cc",
-#     ],
-#     copts = COPTS,
-#     tags = ["team:core"],
-#     deps = [
-#         ":gcs_server_lib",
-#         ":ray_mock",
-#         "@com_google_googletest//:gtest_main",
-#     ],
-# )
+cc_test(
```

Comment:
Is this relevant to this PR?

Reply:
No. But if you ask, I can move it to another PR.

Reply:
I think this is fine, but why is it here? Was it intentional? (I wondered if it has bad merge with the master or sth initially)

Reply:
Yes. I think it's actually bad behavior to mix things not related to this PR here. 

It's intentional... I feel shame about this... Let me remove this change and submit another PR for this.